force_gromacs/pipeline/gromacs_pipeline.py

```diff
@@ -55,9 +68,23 @@ def __getitem__(self, ind):
             return self.named_steps[ind]
         return command
 
-    @property
-    def named_steps(self):
-        return dict(**dict(self.steps))
+    # --------------------
+    #   Private Methods
+    # --------------------
+
+    def _iter(self):
```

Comment:
Can this be a standard python `__iter__` instead?

Reply:
Yes, you are right we could convert this function to an `__iter__` method

Reply:
Also we could probably simplify this method too